examples/se_2_3_localization.cpp

```diff
@@ -0,0 +1,412 @@
+    manif::SE_2_3Tangentd  u_simu, u_est, u_unfilt;
+    Vector9d            u_nom, u_noisy, u_noise;
+    Array9d             u_sigmas;
+    Matrix9d            U;
+
+    u_sigmas << 0.0, 0.0, 0.0, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1;
+    U        = (u_sigmas * u_sigmas).matrix().asDiagonal();
+
+    // Declare the Jacobians of the motion wrt robot and control
+    manif::SE_2_3d::Jacobian J_x, J_u;
+
+    // Define five landmarks in R^3
+    Vector3d b0, b1, b2, b3, b4, b;
+    b0 << 2.0,  0.0,  0.0;
+    b1 << 3.0, -1.0, -1.0;
+    b2 << 2.0, -1.0,  1.0;
+    b3 << 2.0,  1.0,  1.0;
+    b4 << 2.0,  1.0, -1.0;
+    std::vector<Vector3d> landmarks;
+    landmarks.push_back(b0);
+    landmarks.push_back(b1);
+    landmarks.push_back(b2);
+    landmarks.push_back(b3);
+    landmarks.push_back(b4);
+
+    // Define the beacon's measurements
+    Vector3d                y, y_noise;
+    Array3d                 y_sigmas;
+    Matrix3d                R;
+    std::vector<Vector3d>   measurements(landmarks.size());
+
+    y_sigmas << 0.01, 0.01, 0.01;
+    R        = (y_sigmas * y_sigmas).matrix().asDiagonal();
+
+    // Declare the Jacobian of the measurements wrt the robot pose
+    Matrix<double, 3, 9>    H;      // H = J_e_x
+
+    // Declare some temporaries
+    Vector3d                e, z;   // expectation, innovation
+    Matrix3d                E, Z;   // covariances of the above
+    Matrix<double, 9, 3>    K;      // Kalman gain
+    manif::SE_2_3Tangentd      dx;     // optimal update step, or error-state
+    manif::SE_2_3d::Jacobian   J_xi_x; // Jacobian is typedef Matrix
+    Matrix<double, 3, 9>       J_e_xi; // Jacobian
+
+    //
+    //
+    // CONFIGURATION DONE
+
+
+
+    // DEBUG
+    cout << std::fixed   << std::setprecision(3) << std::showpos << endl;
+    cout << "X STATE     :    X      Y      Z    TH_x   TH_y   TH_z     V_x   V_y    V_z" << endl;
+    cout << "---------------------------------------------------------------------------" << endl;
+    cout << "X initial   : " << X_simulation.log().coeffs().transpose() << endl;
+    cout << "---------------------------------------------------------------------------" << endl;
+    // END DEBUG
+
+
+
+
+    // START TEMPORAL LOOP
+    //
+    //
+
+    // Make 10 steps. Measure up to three landmarks each time.
+    for (double t = 0; t < 10; t+=dt)
+    {
+        //// I. Simulation ###############################################################################
+
+        /// get current state and measurements from previous step
+        auto R_k = X_simulation.rotation();
+        auto v_k = X_simulation.linearVelocity();
+        auto acc_k = alpha_prev + R_k.transpose()*g;
+
+        /// input vector
+        u_nom << (dt*(R.transpose())*v_k + 0.5*dt*dt*acc_k),  dt*omega_prev, dt*acc_k;
+
+        /// simulate noise
+        u_noise = u_sigmas * Array9d::Random();             // control noise
+        u_noisy = u_nom + u_noise;                          // noisy control
```

Comment:
Since you used the simulated `R_k` to compute `u_nom`, then `u_nom` is only valid for the simulator.